src/scss/_main.scss

```diff
@@ -1354,6 +1343,160 @@ nav {
   }
 }
 
+.-m-r {
+  margin-right: 0.25rem;
+}
+
+.-m-l {
```

Comment:
i think we were trying to stay away from using stand-alone modifier classes. they should always be chained to a box selector. in reviewing where this is used, we could utilize already established mechanisms by utlizing `.form-controls` and `.form-control` classes, though they might need to be extended?